template-for-proof/README.md

```diff
@@ -8,3 +8,14 @@ To run the proof.
   to your path.
 * Run "make".
 * Open html/index.html in a web browser.
+
+Getting started with Makefiles using `arpa`
```

Comment:
How about "To use `arpa` to simplify writing Makefiles:" to parallel the English phrase that starts the prior paragraph.  

Don't make one of "to run the proof" and "to use arpa" a section header and not the other.  Recommend making them both subsections.